HOWTO

```diff
@@ -45,3 +45,5 @@ The version numbers of the build is defined in ikvm/CommonAssemblyInfo.cs.
 
 If you want to do a strong named build, make sure you have a key container installed named
 "ikvm-key" and run the build with "nant signed".
+
+If the build fails, try to build it using Java 6.
```

Comment:
We can move this in the "Windows:" section or before it as: Make sure you run the build using Java 6

Reply:
Done